### DISCUSSION TITLE
How to tell what container engine rancher desktop is using

### DISCUSSION THREAD

**Discussion by mgingell** [rancher-sandbox/rancher-desktop#3243] [Q&A] [answered]
Without using the UI, is there a way I can tell (from a script) which container engine is being used by rancher desktop?  Specifically, for linux (Ubuntu and WSL).
Thanks,
 - Mike

**Comment by mook-as** [comment, marked as answer]
You should be able to use something like `rdctl list-settings | jq .kubernetes.containerEngine` for that.  Note though we're not currently promising that interface to be stable yet; in particular, we may change the locations of settings in the future (you'll notice that this doesn't _really_ make sense under `kubernetes`).

**Reply by mgingell** [reply]
Too easy - I forgot about rdctl.  Thanks.
